xcodeproj/internal/xcode_schemes.bzl

```diff
@@ -34,6 +43,8 @@ def _focus_schemes(schemes, focused_targets):
             if build_targets:
                 build_action = xcode_schemes_internal.build_action(
                     targets = build_targets,
+                    pre_actions = _focus_actions(build_targets, build_action.pre_actions),
```

Comment:
Since the pre/post actions won't need build targets (we can have a `build_action` with just pre/post actions, and they might not even have a `target` set) this won't work correctly. 

I'll post up a fixed version to the PR before we merge (so you can ignore this, this is a comment for me).

Reply:
Sounds good, thanks!

Reply:
Yeah, that makes sense!